docs/KFSERVING_DEBUG_GUIDE.md

```diff
@@ -108,6 +108,30 @@ Traceback (most recent call last):
 StopIteration
 ```
 
+### Inference Service cannot fetch docker images from AWS ECR
+If you don't see the inference service created at all for custom images from private registries (such as AWS ECR), it might be that the Knative Serving Controller fails to authenticate itself against the registry.
+
+```bash
+failed to resolve image to digest: failed to fetch image information: unsupported status code 401; body: Not Authorized
+```
+
+You can verify that this is actually the case by spinning up a pod that uses your image. The pod should be able to fetch it, if the correct IAM roles are attached, while Knative is not able to. To circumvent this issue you can skip tag resolution.
```

Comment:
@fvde Please help link this [knative doc](https://knative.dev/docs/serving/tag-resolution/) so people know what tag resolution means and what's the implication if you turn that off, thanks!

Reply:
Done, thanks for the feedback! 👍